src/styles/blocks/card.scss

```diff
@@ -0,0 +1,120 @@
+.card {
+  width: 198px;
+  height: 406px;
+  background-color: $white-color;
+  border: 1px solid $gray-color;
+  border-radius: 5px;
+  box-sizing: content-box;
+  transition: all 0.3s;
```

Comment:
typically it is better to target particular properties instead of just using `all`